Library/Formula/libdistance.rb

```diff
@@ -0,0 +1,12 @@
+class Libdistance < Formula
+  homepage "http://monkey.org/~jose/software/libdistance/"
+  url "http://monkey.org/~jose/software/libdistance/libdistance-0.2.2.tar.gz"
+  sha1 "d516e5f24ae55bcbbf93a4d14c6a53914e5b3a0a"
+
+  def install
+    # The build system is incredibly simple.
+    system "make"
+    include.install "distance.h"
+    lib.install "libdistance.a"
+  end
+end
```

Comment:
libdistance actually ships with a test suite, but it seems that the original sources are gone by the time that `Libdistance.test` is run. If we build the test program, can it be installed to the Cellar but not /usr/local/bin?